tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile.py

```diff
@@ -253,3 +198,38 @@ def test_desandbox_fn(self):
     self.assertEqual(desandbox('/some/path/.pants.d/tmp.pants.d/cool/beans'), '.pants.d/tmp.pants.d/cool/beans')
     self.assertEqual(desandbox('/some/path/.pants.d/exec-location/.pants.d/tmp.pants.d/cool/beans'),
                                '.pants.d/tmp.pants.d/cool/beans')
+
+  def wrap_in_vts(self, invalid_targets):
+    return [LightWeightVTS(t) for t in invalid_targets]
+
+  def init_dependencies_for_scala_libraries(self):
+    init_subsystem(
+      ScalaPlatform,
+      {
+        ScalaPlatform.options_scope: {
+          'version': 'custom',
+          'suffix_version': '2.12',
+        }
+      }
+    )
+    self.make_target(
+      '//:scala-library',
+      target_type=JarLibrary,
+      jars=[JarDependency(org='com.example', name='scala', rev='0.0.0')]
+    )
+
+  def create_task_with_target_roots(self, target_roots):
+    context = self.context(target_roots=target_roots)
+    self.init_products(context)
+    task = self.create_task(context)
+    # tried for options, but couldn't get it to reconfig
```

Comment:
Is this referring to not being able to set the size estimator using a task option?

Reply:
yes.